app/controllers/fact_values_controller.rb

```diff
@@ -5,13 +5,7 @@ class FactValuesController < ApplicationController
   before_action :setup_search_options, :only => :index
 
   def index
-    base = resource_base
-    begin
-      values = base.my_facts.search_for(params[:search], :order => params[:order])
-    rescue => e
```

Comment:
In my opinion this error handling removal is OK - it's from 2011 and I'd prefer not to hide potential exceptions